crates/swc_common/src/syntax_pos.rs

```diff
@@ -505,39 +505,7 @@ impl Span {
 
     #[inline]
     pub fn private(self) -> Span {
-        self.apply_mark(Mark::fresh(Mark::root()))
-    }
-
-    #[inline]
-    pub fn adjust(&mut self, expansion: Mark) -> Option<Mark> {
```

Comment:
These functions are never used through our codebase, according to the function name, I think it is more related to macro. In our context, we may don't have such a complex scenario, so delete it.

Reply:
cc @kwonoj thoughts? Do you think we can delete this?

Reply:
What was the original use case? I think it's ok if there's no usage at all - but if there are legit usecase, even though it's not being used internally it may need to be preserved. swc/core is not the only consumer of this module, as this is a public interface.

Reply:
I didn't use this. I simply copied whole module from rustc